homeassistant/components/plaato/config_flow.py

```diff
@@ -30,7 +30,7 @@
 class PlaatoConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     """Handles a Plaato config flow."""
 
-    VERSION = 2
+    VERSION = 1
```

Comment:
Is the version change intended?

Reply:
Yes, I wasn't sure how that was used but realized that I need some kind of migration handler to satisfy the version bump. Since no migration should be needed there's no need to bump it.